CHARTER.md

```diff
@@ -1,10 +1,11 @@
-# Open Container Initiative Charter v1.0
+# Open Container Initiative Charter v1.1
 
 Effective 13 November 2015
+Updated 6 May 2020
 
 ##  1. Mission of the Open Container Initiative (“OCI”).
 
-The Open Container Initiative provides an open source technical community within which industry participants may easily contribute to building a vendor-neutral, portable and open specification and runtime that deliver on the promise of containers as a source of application portability backed by a certification program.
+The Open Container Initiative provides an open source technical community within which industry participants may easily contribute to building vendor-neutral, portable and open specifications and runtime that deliver on the promise of containers as a source of application portability backed by a certification program.
```

Comment:
I agree with @samuelkarp and we should make `runtime` definition clear here. We have more discussion about scope, but right now, runtime spec is the key and image/distribution spec are helping to delivery runtime bundle fast and security.  

IMO, if we just extend distribution spec without thinking about `runtime`, it sounds like we try to build storage spec unrelated to `container`. And people will treat registry as other kind of `object storage service`. I think it is not the goal of `opencontainers`. Make senses?

Reply:
Are these mutual exclusive (runtime spec definition, and having distribution be more than just runnable images)? The work we're doing around the distribution spec has an economy of scale. What we do for any artifact that can be stored in a distribution spec equally applies to container images (Notary v2, SBOM efforts, search, pub/sub, extensibility model).
If we focus on a broader ecosystem of serving content, there's more interest and support for adding functionality. 
In the end, doesn't image distribution and the OCI become more interesting and relevant?

Reply:
I believe that (as usual), the "and runtime" was probably just a carve-out for `runc`. As discussed in the call, I think we need to rethink how we describe the OCI fairly fundamentally -- I like the idea of it being described place for "boring white-label infrastructure" to empower the container ecosystem.